iceoryx_utils/test/moduletests/test_semaphore_module.cpp

```diff
@@ -323,10 +316,10 @@ TIMING_TEST_P(Semaphore_test, TimedWaitWithTimeout, Repeat(3), [&] {
     std::atomic_bool timedWaitFinish{false};
 
     std::thread t([&] {
-        auto ts = Duration::fromNanoseconds(TIMING_TEST_TIMEOUT).timespec(TimeSpecReference::Epoch);
+        auto ts = Duration::fromNanoseconds(TIMING_TEST_TIMEOUT);
```

Comment:
nitpicking, this isn't a `ts` anymore ;)

Reply:
upsi